ribasim_qgis/widgets/dataset_widget.py

```diff
@@ -93,8 +99,9 @@ def remove_geopackage_layers(self) -> None:
             return
 
         # Start deleting
-        elements = {item.element for item in selection}
+        elements = {item.element for item in selection}  # type: ignore
```

Comment:
This needs to be fixed. `item.element` is dynamically added and is not part of the variables in `item`. Therefore we should change this. Maybe just with a simple dictionary in `self` to keep track of the elements per item.

Reply:
I've added TODOs